modular_zubbers/code/game/objects/items/holy_weapons.dm

```diff
@@ -0,0 +1,36 @@
+/obj/item/nullrod/papal_staff
+	name = "papal staff"
+	desc = "A staff used by traditional bishops and popes."
+	icon = 'modular_zubbers/icons/obj/items_and_weapons.dmi'
+	icon_state = "papal_staff"
+	inhand_icon_state = "papal_staff"
+	lefthand_file = 'modular_zubbers/icons/mob/inhands/weapons/melee_lefthand.dmi'
+	righthand_file = 'modular_zubbers/icons/mob/inhands/weapons/melee_righthand.dmi'
+	w_class = WEIGHT_CLASS_BULKY
+	attack_verb_continuous = list("smacks", "strucks", "cracks", "beats", "purifies")
+	attack_verb_simple = list("smack", "struck", "crack", "beat", "purify")
+	slot_flags = 0
```

Comment:
Can be removed again once I do some CI changes.

Reply:
I'm not sure what you mean by that, sorry! Should I remove these lines?

Reply:
Try removing the slot_flags line now, I pushed a fix for CI.